app/i18n/locales/en.json

```diff
@@ -565,6 +565,8 @@
   "Unsupported_system_message": "Unsupported system message",
   "Updating": "Updating...",
   "Uploading": "Uploading",
+  "Error_on_upload_archive": "Error on upload archive",
```

Comment:
This might be a false cognate or only feel wrong.
It should be `Error on file upload`
Did you search for this on the main repo?